### PR TITLE
Report test stats for macos_10_13 tests

### PR DESCRIPTION
**Pull request by janeyx99**
Run print_test_stats.py for macos_10_13 tests.

Test plan:
Make sure CI passes, specifically for macos_10_13

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1417,6 +1417,25 @@ jobs:
...
+            export SCRIBE_GRAPHQL_ACCESS_TOKEN="${SCRIBE_GRAPHQL_ACCESS_TOKEN}"
+            export CIRCLE_TAG="${CIRCLE_TAG:-}"
+            export CIRCLE_SHA1="$CIRCLE_SHA1"
+            export CIRCLE_PR_NUMBER="${CIRCLE_PR_NUMBER:-}"
+            export CIRCLE_BRANCH="$CIRCLE_BRANCH"
+            export CIRCLE_JOB="$CIRCLE_JOB"
+            export CIRCLE_WORKFLOW_ID="$CIRCLE_WORKFLOW_ID"
+            export AWS_ACCESS_KEY_ID=${CIRCLECI_AWS_ACCESS_KEY_FOR_WIN_BUILD_V1}
+            export AWS_SECRET_ACCESS_KEY=${CIRCLECI_AWS_SECRET_KEY_FOR_WIN_BUILD_V1}
+            export PYTHONPATH="$PWD"
```
Comment (seemethere) [inline .circleci/config.yml:1435]: Are most of these environment variables actually necessary?

Seems to me like the `CIRCLE_*` variables should already be defined in the variables and things like `PYTHONPATH` should just be set by the python interpreter itself.

Comment (janeyx99) [reply .circleci/config.yml:1435]: Hm, they were for the Windows runners but I'll make a commit removing them and seeing if that works

**Comment by janeyx99** [comment]
Closing in favor of #56429

File: .circleci/config.yml (hunk 1 above)
Comment (walterddr) [reply .circleci/config.yml:1435]: `PYTHONPATH` is set there to ensure git repo root is in front of the site-package/torch.
I think I only tested this in a linux docker container env. not sure if it is still necessary in win/mac

Comment (walterddr) [reply .circleci/config.yml:1435]: also I think you should use `export PYTHONPATH="\${PWD}"`

**Pull request closed without merging**